src/axom/mint/docs/sphinx/sections/synopsis.rst

```diff
@@ -40,12 +40,12 @@ the following third-party libraries is provided:
 
 * `RAJA`_, used for the parallel execution and portability layer.
 
-* `Conduit`_ , for using `Sidre`_ as the :ref:`MeshStorageManagement` system.
+* `Conduit`_ , for using `Sidre <../../../sidre/docs/sphinx/index.html>`_ as the :ref:`MeshStorageManagement` system.
```

Comment:
I think this file (mint's sphinx synopsis.rst) was replaced by index.rst.  It doesn't look like synopsis.rst is linked to from anywhere, and should be removed.  I realized this when I looked at the warnings that came up when this file was built.  Thoughts, @bmhan12 and @gzagaris ?

Reply:
@agcapps Based off the [raw text](https://raw.githubusercontent.com/LLNL/axom/develop/src/axom/mint/docs/sphinx/index.rst) for mint's `index.rst`, `index.rst` links to `synopsis.rst`. It's not visible for some reason in Github's [preview](https://github.com/LLNL/axom/blob/develop/src/axom/mint/docs/sphinx/index.rst) view of the file.

Reply:
That's right `synopsis.rst` is included by the `index.rst` -- it consists of the short summary for each component that gets rendered on the component's landing page.

Reply:
Aha!  Good reason not to delete the file.  Sorry I didn't see that immediately, George.